vNext/test/TestExtensions/TestExtensions.csproj

```diff
@@ -48,6 +48,9 @@
     <Compile Include="..\..\..\test\TestExtensions\TestDefaultConfiguration.cs">
       <Link>TestDefaultConfiguration.cs</Link>
     </Compile>
+    <Compile Include="..\..\..\test\TestExtensions\TesterUtils.cs">
+      <Link>TesterUtils.cs</Link>
+    </Compile>
```

Comment:
what's this for? Doesn't this have stuff related to azure storage or utilities that are really not very general and bring extra baggage with them? But even without the extra baggage, what's the reason for moving it up the hierarchy? Tester and TesterInternal currently hold a lot of unrelated functionality for testing every single project in the solution, but remember to try to keep the new test projects and TestExtensions free of all these, and more to the point.